src/shared-components/OrganizationCard/OrganizationCard.tsx

```
@@ -76,246 +78,248 @@ function OrganizationCard({
+              )}
+            </div>
+            <div className={styles.content}>
+              <div>
+                {/* Tooltip for the organization name */}
+                <Tooltip title={name} placement="top-end">
+                  <h4 className={[styles.orgName, 'fw-semibold'].join(' ')}>
+                    {name}
+                  </h4>
+                </Tooltip>
+                <output
+                  role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)